driver_test.go

```diff
@@ -469,6 +470,47 @@ func runningOnGCP() bool {
 	return os.Getenv("CLOUD_PROVIDER") == "GCP"
 }
 
+func TestBogusUserPasswordParameters(t *testing.T) {
+	// Different error message is returned depending on whether the user is known or not.
+	var invalidDNS string
+	if runningOnGithubAction() {
+		invalidDNS = fmt.Sprintf("%s:%s@%s", "bogus", pass, host)
+		invalidUserPassErrorTests(invalidDNS, 390422, t)
```

Comment:
Can we split it into two separate tests? Now we have one test which actually invokes two internally.